elmslie-android/src/main/java/vivid/money/elmslie/android/renderer/ElmRenderer.kt

```diff
@@ -149,7 +149,12 @@ internal class ElmRenderer<Effect : Any, State : Any>(
                         val list = mapListItems(state)
                         state to list
                     }
-                    .catch { logger.fatal("Crash while mapping state", it) }
+                    .catch {
+                        logger.fatal(
+                            message = "Crash while mapping state",
```

Comment:
maybe we should add store key to store interface, in this case we could add tags here as well